### DISCUSSION TITLE
volume processing time

### DISCUSSION THREAD

**Discussion by deyamyot** [biigle/core#735] [Q&A] [answered]
Hi, i'm trying to create new volumes in my project with some newly uploaded videos and even if it's been a couples of days, when i clic on the videos i still have the **'The video has not been processed yet. Please try again later'** message. I tryed to use different videos and to reupload the videos in the storage. I was wonderring if something have change since the january modification of BIIGLE for the processing time and i'm just impatient or is it a problem off corruption on my end. Thank you.

**Comment by mzur** [comment, marked as answer]
Hi, yes we are aware of this issue. BIIGLE is currently processing a couple of thousand new videos which are hosted remotely with a very slow connection, so processing takes ages. We can't isolate these videos so everything else also has to wait for the processing. We'll try to find a solution for cases like these but right now there is no quick fix, sorry.

**Comment by deyamyot** [comment]
Hi,
I understand. I will wait then.

Thank you for the quick respond.

Cheers,
Audrey
________________________________

**Comment by deyamyot** [comment]
Oh and do you have an estimation time of when thoses big videos will be process? So that i have an idea of how much its gonna delayed my project

Thank you
Audrey
________________________________

**Reply by mzur** [reply]
We'll monitor the situation over the weekend. If the issue is not resolved on Monday, we'll try to find a fix. In the worst case, we'll have to kill all processing jobs and restart them manually. In this case, all new videos with a decent connection (including uploaded videos) should be available by Tuesday or Wednesday.

**Comment by deyamyot** [comment]
Good, thank you

Have a nice day
Audrey

De : Martin Zurowietz ***@***.***>
Envoyé : 19 janvier 2024 14:21
À : biigle/core ***@***.***>
Cc : Audrey-Anne Amyot ***@***.***>; Author ***@***.***>
Objet : Re: [biigle/core] volume processing time (Discussion #735)


We'll monitor the situation over the weekend. If the issue is not resolved on Monday, we'll try to find a fix. In the worst case, we'll have to kill all processing jobs and restart them manually. In this case, all new videos with a decent connection (including uploaded videos) should be available by Tuesday or Wednesday.

-

**Reply by mzur** [reply]
We have implemented a temporary fix and your videos should be available now. We'll work on a permanent solution.

**Comment by deyamyot** [comment]
Yes, it works. Thank you

**Comment by ilabardi** [comment]
Hi! 
Your post was from over a year ago but I am having the same problem now! My videos are about 14 minutes long. I tried uploading 4 of them in a volume to annotate but having the same message "The video has not been processed. Try again later". Not sure what to do or how long to wait! 
Thank you,

**Reply by mzur** [reply]
Please check again. There is a backlog of processing jobs again, which causes the delays.

**Reply by ilabardi** [reply]
Yes-everything is working now! Thank you!
